ERROR - 2023-08-04 22:18:01 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(174): mysqli->real_connect()
#1 /home/documentation/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/documentation/app/Controllers/Accueil.php(39): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
#4 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
#5 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#6 /home/documentation/public/index.php(67): CodeIgniter\CodeIgniter->run()
#7 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /home/documentation/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/documentation/app/Controllers/Accueil.php(39): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
#3 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
#4 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#5 /home/documentation/public/index.php(67): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2023-08-04 22:18:01 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 APPPATH/Controllers/Accueil.php(39): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
 3 SYSTEMPATH/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-04 22:19:43 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(174): mysqli->real_connect()
#1 /home/documentation/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/documentation/app/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
#4 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
#5 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#6 /home/documentation/public/index.php(67): CodeIgniter\CodeIgniter->run()
#7 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /home/documentation/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/documentation/app/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
#3 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
#4 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#5 /home/documentation/public/index.php(67): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2023-08-04 22:19:43 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 APPPATH/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
 3 SYSTEMPATH/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-04 22:19:43 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(174): mysqli->real_connect()
#1 /home/documentation/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/documentation/app/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
#4 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
#5 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#6 /home/documentation/public/index.php(67): CodeIgniter\CodeIgniter->run()
#7 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /home/documentation/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/documentation/app/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
#3 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
#4 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#5 /home/documentation/public/index.php(67): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2023-08-04 22:19:43 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 APPPATH/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
 3 SYSTEMPATH/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-04 22:19:44 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(174): mysqli->real_connect()
#1 /home/documentation/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/documentation/app/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
#4 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
#5 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#6 /home/documentation/public/index.php(67): CodeIgniter\CodeIgniter->run()
#7 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /home/documentation/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/documentation/app/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
#3 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
#4 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#5 /home/documentation/public/index.php(67): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2023-08-04 22:19:44 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 APPPATH/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
 3 SYSTEMPATH/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-04 22:21:10 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(174): mysqli->real_connect()
#1 /home/documentation/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/documentation/app/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
#4 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
#5 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#6 /home/documentation/public/index.php(67): CodeIgniter\CodeIgniter->run()
#7 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /home/documentation/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/documentation/app/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
#3 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
#4 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#5 /home/documentation/public/index.php(67): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2023-08-04 22:21:10 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 APPPATH/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
 3 SYSTEMPATH/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-04 22:21:12 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(174): mysqli->real_connect()
#1 /home/documentation/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/documentation/app/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
#4 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
#5 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#6 /home/documentation/public/index.php(67): CodeIgniter\CodeIgniter->run()
#7 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /home/documentation/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/documentation/app/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
#3 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
#4 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#5 /home/documentation/public/index.php(67): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2023-08-04 22:21:12 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 APPPATH/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
 3 SYSTEMPATH/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-04 22:22:06 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(174): mysqli->real_connect()
#1 /home/documentation/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/documentation/app/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
#4 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
#5 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#6 /home/documentation/public/index.php(67): CodeIgniter\CodeIgniter->run()
#7 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /home/documentation/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/documentation/app/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
#3 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
#4 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#5 /home/documentation/public/index.php(67): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2023-08-04 22:22:06 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 APPPATH/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
 3 SYSTEMPATH/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-04 22:22:23 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(174): mysqli->real_connect()
#1 /home/documentation/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/documentation/app/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
#4 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
#5 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#6 /home/documentation/public/index.php(67): CodeIgniter\CodeIgniter->run()
#7 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /home/documentation/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/documentation/app/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
#3 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
#4 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#5 /home/documentation/public/index.php(67): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2023-08-04 22:22:23 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 APPPATH/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
 3 SYSTEMPATH/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-08-04 22:22:38 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:174
Stack trace:
#0 /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php(174): mysqli->real_connect()
#1 /home/documentation/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#2 /home/documentation/app/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
#3 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
#4 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
#5 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#6 /home/documentation/public/index.php(67): CodeIgniter\CodeIgniter->run()
#7 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in /home/documentation/vendor/codeigniter4/framework/system/Database/MySQLi/Connection.php:218
Stack trace:
#0 /home/documentation/vendor/codeigniter4/framework/system/Database/BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect()
#1 /home/documentation/app/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
#2 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
#3 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
#4 /home/documentation/vendor/codeigniter4/framework/system/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
#5 /home/documentation/public/index.php(67): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2023-08-04 22:22:38 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH/Database/BaseConnection.php on line 427.
 1 APPPATH/Controllers/Accueil.php(42): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH/CodeIgniter.php(904): App\Controllers\Accueil->__construct()
 3 SYSTEMPATH/CodeIgniter.php(488): CodeIgniter\CodeIgniter->createController()
 4 SYSTEMPATH/CodeIgniter.php(366): CodeIgniter\CodeIgniter->handleRequest()
 5 FCPATH/index.php(67): CodeIgniter\CodeIgniter->run()
